services/121-service/src/migration/1724754547895-modify-safaricom-requests-table.ts

```diff
@@ -58,21 +58,6 @@ export class ModifySafaricomRequestsTable1724754547895
       `ALTER TABLE "121-service"."safaricom_request" RENAME COLUMN "originatorConversationID" TO "originatorConversationId"`,
     );
 
-    await queryRunner.query(
-      `ALTER TABLE "121-service"."safaricom_request" DROP CONSTRAINT "FK_76a85e893850c2ef7ce4b6441a0"`,
-    );
-    await queryRunner.query(
-      `ALTER TABLE "121-service"."safaricom_request" ADD CONSTRAINT "FK_1f9a3e9003d584d12009b19b568" FOREIGN KEY ("mpesaTransactionId") REFERENCES "121-service"."transaction"("id") ON DELETE NO ACTION ON UPDATE NO ACTION`,
-    );
-
-    await queryRunner.query(
-      `ALTER SEQUENCE IF EXISTS "121-service".safaricom_request_id_seq RENAME TO safaricom_transfer_id_seq;`,
-    );
-
-    await queryRunner.query(
-      `ALTER TABLE "121-service"."safaricom_request" ALTER COLUMN "id" SET DEFAULT nextval('"121-service"."safaricom_transfer_id_seq"')`,
-    );
-
```

Comment:
Why do we remove these?

Reply:
@ashishkumarbasak-accenture sorry, that must have gone wrong in merging. I thought you removed this on your side. Can you put back the relevant lines? Thanks.